addon/components/hyper-table/cell-renderers/money.js

```diff
@@ -1,10 +1,9 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { empty } from '@ember/object/computed';
+import EditableMixin from '../../../mixins/editable';
 
-export default Component.extend({
-  tagName: '',
```

Comment:
removing the `tagName` is ok ?

Reply:
when the tagName is an empty string I get this error `Assertion Failed: You cannot access this.$() on a component with tagName: '' specified.`
When it's either removed or not empty, the error is gone.